### PR TITLE
Add basic UI

### PR DESCRIPTION
**Pull request by cilfonegabriel**
# Dear Code Reviewers.
### In this project:

- Create a `app.rb` file that will serve as your console app entry-point. It should have methods that do the following:
  - List all books.
  - List all people.
  - Create a person (teacher or student, not a plain `Person`).
  - Create a book.
  - Create a rental.
  - List all rentals for a given person id.
- In your `main.rb` define the entry point, this will be a method called `main` that is invoked at the end of your file. This method should do the following:
  - Present the user with a list of options to perform.
  - Lets users choose an option.
  - If needed, ask for parameters for the option.
  - Have a way to quit the app.

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -1,25 +1,63 @@
...
+  option = -1
+  app = App.new
+  until option.zero?
+    main_menu
+    option = gets.chomp.to_i
+    puts ''
+    run_menu(app, option) unless option.zero?
+  end
+  puts 'Thank you for using this app'
+end
```
Comment (pinkmoon25) [inline main.rb:63]: - [ ] Good job creating the console UI, however please notice that you have forgotten to call your main function at the end, currently when you run the `ruby main.rb` command nothin happens, call it at the end like: `main()`

File: main.rb [hunk 2]
```diff
@@ -1,25 +1,63 @@
...
+  when 2
+    app.list_people
+  when 3
+    app.create_book
+  when 4
+    add_person(app)
+  when 5
+    app.create_rental
+  when 6
+    list_rentals_per_person_id(app)
```
Comment (pinkmoon25) [inline main.rb:47]: - [x] Your option 6 list rentals is not working correctly, currently you have to call it as app.list_rental_per_person, just like you have called the other methods, also notice the typo 's' in the method name.
- [x] You will have to correct the method also inside app.rb to ask for id or any other input that you see fit for the list rental option, currently when you select 6 it won't do anything

File: library/app.rb [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+  def create_rental
+    puts 'Select a person from the list by number (not ID)'
+    @people.each_with_index do |person, i|
+      puts(" #{i}) [#{person.class}] ID: #{person.id} Name: #{person.name} Age: #{person.age}")
+    end
+    index = gets.chomp.to_i
+    person = @people[index]
+    puts 'Select a book from the list by number'
+    @book.each_with_index { |book, i| puts(" #{i}) | Title: #{book.title} Author: #{book.author} |") }
```
Comment (pinkmoon25) [inline library/app.rb]: - [ ] I think here it should be @books and not @book, this is throwing an error in the console, please fix it

**Review by pinkmoon25 — CHANGES_REQUESTED** [review]
Hi @cilfonegabriel , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
Rubocop checks :heavy_check_mark: 
gitflow :heavy_check_mark: 
Implemented Ui for app :heavy_check_mark: 
Create book and person :heavy_check_mark: 
List books and people :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

**[OPTIONAL] Now that our app is a console app, I suggest you add the instructions in your Readme file as to how a normal user can also run the app locally, this will definitely help showcase your documentation skills too :+1: , for the most part you have already explained but also mention the command `ruby main.rb` which enables the user to use the app on console**

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @cilfonegabriel, 


68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 

✅ Descriptive PR 👍 
✅ Nice UI interaction🔥 
✅ Console app has the correct functionality 🔥 
✅ Professional README file 💯 
✅ Linters are passing 👏 


## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**